backend/package.json

```
@@ -4,7 +4,7 @@
   "description": "PBCEx Backend API Server",
   "main": "dist/server.js",
   "engines": {
```

Biome (2.1.2)

[error] The key engines was already declared.

This where a duplicated key was declared again.

If a key is defined multiple times, only the last definition takes effect. Previous definitions are ignored.

(lint/suspicious/noDuplicateObjectKeys)